### DISCUSSION TITLE
Divider support of tailwind colors

### DISCUSSION THREAD

**Discussion by mark1502** [saadeghi/daisyui#2767] [Ideas / feature requests]
It looks like I can change the color of a divider by adding a class like 'divider-info' or 'divider-primary'.  But it would be great if I could also specify a tailwind color, like 'divider-blue-600'.  That would allow for a greater spectrum of choices.

It would also be nice if I could specify the height (thickness) of the line.

Thanks!

**Comment by saadeghi** [comment]
You can do this:  
https://play.tailwindcss.com/GEAImB7Aqd
```html
<div class="divider before:bg-red-500 after:bg-red-500">red</div>
<div class="divider before:h-2 after:h-2">height</div>
```

Let me know if you have a question.

**Comment by mark1502** [comment]
That's great, thanks!  I was also able to combine the attributes like this:
```
<div class="divider before:bg-red-500 before:h-2 after:bg-red-500 after:h-2">red</div>
```
Thanks again!
